api/introduction.asciidoc

```diff
@@ -105,3 +105,20 @@ When the _revision_ number changes there may be an impact on the behavior of
 existing functionality, but this should not affect backwards compatibility.
 Functionality should not be added or removed when the _revision_ number
 changes.
+
+[[unified-spec]]
+== Unified Specification
+
+This document specifies all versions of the OpenCL API.
+
+There are three ways that an OpenCL feature may be described in terms of what
+versions of OpenCL support that feature.
+
+  * Missing before _major.minor_: Features that were introduced in
+    version _major.minor_.  Implementations of an earlier version of OpenCL
+    will not provide these features.
+  * Deprecated by _major.minor_: Features that were deprecated
```

Comment:
Does it make sense to add something like `Missing after` or `Removed by`? Just to be able to clearly state when particular feature was removed

Reply:
Yes, I think that would make sense.  The problem is that OpenCL doesn't seem to have explicitly removed anything.  I have created an issue to consider this point: https://github.com/KhronosGroup/OpenCL-Docs/issues/153